src/navigation/nav2_simple_navigator/src/simple_navigator.cpp

```diff
@@ -12,11 +12,6 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// Navigation Strategy based on:
```

Comment:
Why remove this? I figured a reference to the paper would be a very useful comment.

Reply:
Wrong place for it. This module (SimpleNavigator) is not implementing any particular algorithm. It is simply invoking a component that computes and path and then another that follows the path. We should move the comment to the appropriate place.